mobile/src/components/Settings.jsx

```diff
@@ -13,7 +13,7 @@ const SubCategory = ({ id, label, value, title }) => (
   </div>
 )
 
-export const Settings = ({ t, version, serverUrl, backupImages, setBackupImages, showUnlinkConfirmation, displayUnlinkConfirmation, hideUnlinkConfirmation, unlink }) => (
+export const Settings = ({ t, version, serverUrl, backupImages, setBackupImages, showUnlinkConfirmation, displayUnlinkConfirmation, hideUnlinkConfirmation, unlink, mediaUploading, launchBackup }) => (
```

Comment:
Reading this, I start to consider it will be a good thing to connect SubSettings component rather than the whole Settings component. Do you agree?

Note: it is not necessary to include this re-factorization in this exact PR.